.github/workflows/e2e_ios.yml

```diff
@@ -9,17 +9,8 @@ on:
     branches:
       - 'dependabot/**'
   workflow_dispatch:
-    inputs:
-      run_testRail:
-        description: Run_TestRail
-        type: boolean
-        required: false
```

Comment:
Looks like this was accidentally removed?

Reply:
Added back in